views/options/profile.jade

```diff
@@ -213,6 +213,15 @@ script(id='partials/options.profile.profile.html', type='text/ng-template')
         button.btn.btn-default(ng-click='_editing.profile = true', ng-show='!_editing.profile')= env.t('edit')
         h4=env.t('displayName')
         span(ng-show='profile.profile.name') {{profile.profile.name}}
+        p
+        small
```

Comment:
It was meant to be indented. I was using the p to break the line because that's how it was done in some other part of the code, also also because I just view it as the built-in space after a paragraph in word processors. Should I change it to <code>br</code>, or is it fine just using <code>p</code>?